src/ResourceManagement/Compute/Microsoft.Azure.Management.V2.Compute/MissingNetworkTypes.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
This is a temporary file containing the dummy def for interfaces in the network those are not yet available but required for VMSS. Once those interfaces are available this file will be removed